astroquery/jplhorizons/core.py

```diff
@@ -144,7 +144,7 @@ def ephemerides_async(self, airmass_lessthan=99,
                           rate_cutoff=None,
                           skip_daylight=False,
                           refraction=False,
-                          refsystem='J2000',
+                          refsystem='ICRF',
```

Comment:
Given that J2000 is almost the same as ICRF I suppose this change of default doesn't need to be mentioned in the changelog, but if it's significant for SSO please add a note about it in the changelog, too.

Reply:
It is probably not significant, but we can document the change.  I made the change because J2000 is not listed as an option, in new API documentation.